### PR TITLE
GCP Actions. Load balancer - SSL Policies. Delete

### PR DESCRIPTION
**Pull request by evhenyshaliov**
Added
- implemetation
- docstrings
- example
- unit test

### CODE REVIEW

File: tools/c7n_gcp/tests/test_loadbalancer.py [hunk 1]
```diff
@@ -155,6 +155,34 @@ def test_loadbalancer_ssl_policy_get(self):
         self.assertEqual(instance['kind'], 'compute#sslPolicy')
         self.assertEqual(instance['name'], 'newpolicy')
 
+    def test_loadbalancer_ssl_policy_delete(self):
+        project_id = 'custodian-test-project-0'
+        session_factory = self.replay_flight_data('lb-ssl-policy-delete',
+                                                  project_id=project_id)
+        base_policy = {'name': 'lb-ssl-policy-delete',
+                       'resource': 'gcp.loadbalancer-ssl-policy'}
+
```
Comment (jondavidi) [inline tools/c7n_gcp/tests/test_loadbalancer.py]: I think there should be a check showing how many resources there are before the delete and that they have the invalid TLS version, to verify the correct number of resources was deleted

Comment (jondavidi) [reply tools/c7n_gcp/tests/test_loadbalancer.py]: actually I take this back, I think it's fine the way it is. you're showing the resouce that was deleed, then showing after that there are resources of same type that weren't. disregard this

File: docs/source/gcp/examples/loadbalancer-ssl-policy-delete-old-versions.rst [hunk 2]
```diff
@@ -0,0 +1,17 @@
+Load Balancer - SSL Policies - Delete policies by TLS version
+==============================================================
+
+It's possible to delete all SSL Policies that don't have TLS version 1.2.
+
+.. code-block:: yaml
+
+        policies:
+        - name: gcp-load-balancing-ssl-policies-delete
```
[the remark this answers is not in the document]
Comment (evhenyshaliov) [reply docs/source/gcp/examples/loadbalancer-ssl-policy-delete-old-versions.rst]: Updated

File: tools/c7n_gcp/c7n_gcp/resources/loadbalancer.py [hunk 3]
```diff
@@ -113,6 +114,36 @@ def get(client, resource_info):
                 'sslPolicy': resource_info['name']})
 
 
+@LoadBalancingSslPolicy.action_registry.register('delete')
+class LoadBalancingSslPolicyDelete(MethodAction):
+    """The action is used for Load Balancing SSL Policies delete.
+    GCP resource is https://cloud.google.com/compute/docs/reference/rest/v1/sslPolicies.
```
[the remark this answers is not in the document]
Comment (evhenyshaliov) [reply tools/c7n_gcp/c7n_gcp/resources/loadbalancer.py]: Updated

File: tools/c7n_gcp/c7n_gcp/resources/loadbalancer.py [hunk 4]
```diff
@@ -113,6 +114,36 @@ def get(client, resource_info):
...
+    GCP action is https://cloud.google.com/compute/docs/reference/rest/v1/sslPolicies/delete.
+
+    Example:
+
+    .. code-block:: yaml
+
+        policies:
```
[the remark this answers is not in the document]
Comment (evhenyshaliov) [reply tools/c7n_gcp/c7n_gcp/resources/loadbalancer.py]: Updated

**Pull request closed without merging**